locales/en-US/app.ftl

```diff
@@ -290,6 +295,15 @@ Home--additional-content-content = You can <strong>drag and drop</strong> a prof
 Home--compare-recordings-info = You can also compare recordings. <a>Open the comparing interface.</a>
 Home--your-recent-uploaded-recordings-title = Your recent uploaded recordings
 
+# We replace the elements such as <perf> and <simpleperf> with links to the
+# documentation to use these tools.
+Home--load-files-from-other-tools =
+    The Firefox Profiler can also import profiles from other profilers, such as
```

Comment:
```suggestion
    The { -profiler-brand-name } can also import profiles from other profilers, such as
```

Reply:
aaaw I should have found that myself, thanks for the review, I changed it.